pvlib/iotools/solaranywhere.py

```diff
@@ -0,0 +1,285 @@
+
+    payload = {
+        "Sites": [{
+            "Latitude": latitude,
+            "Longitude": longitude
+        }],
+        "Options": {
+            "OutputFields": variables,
+            "SummaryOutputFields": [],  # Do not request summary/monthly data
+            "SpatialResolution_Degrees": spatial_resolution,
+            "TimeResolution_Minutes": time_resolution,
+            "WeatherDataSource": source,
+            "MissingDataHandling": missing_data,
+        }
+    }
+
+    if true_dynamics:
+        payload['Options']['ApplyTrueDynamics'] = True
+
+    if probability_of_exceedance is not None:
+        if type(probability_of_exceedance) != int:
+            raise ValueError('`probability_of_exceedance` must be an integer')
+        payload['Options']['ProbabilityOfExceedance'] = \
+            probability_of_exceedance
+
+    # Add start/end time if requesting non-TMY data
+    if (start is not None) or (end is not None):
+        # start/end are required to have an associated time zone
+        if start.tz is None:
+            start = start.tz_localize('UTC')
+        if end.tz is None:
+            end = end.tz_localize('UTC')
+        payload['Options']["StartTime"] = start.isoformat()
+        payload['Options']["EndTime"] = end.isoformat()
+
+    # Convert the payload dictionary to a JSON string (uses double quotes)
+    payload = json.dumps(payload)
+    # Make data request
+    request = requests.post(url+'/WeatherData', data=payload, headers=headers)
+    # Raise error if request is not OK
+    if request.ok is False:
+        raise ValueError(request.json()['Message'])
+    # Retrieve weather request ID
+    weather_request_id = request.json()["WeatherRequestId"]
+
+    # The SolarAnywhere API is asynchronous, hence a second request is
+    # necessary to retrieve the data (WeatherDataResult).
+    start_time = time.time()  # Current time in seconds since the Epoch
+    # Attempt to retrieve results until the max response time has been exceeded
+    while True:
+        results = requests.get(url+'/WeatherDataResult/'+weather_request_id, headers=headers)  # noqa: E501
+        results_json = results.json()
+        if results_json.get('Status') == 'Done':
+            if results_json['WeatherDataResults'][0]['Status'] == 'Failure':
+                raise RuntimeError(results_json['WeatherDataResults'][0]['ErrorMessages'][0]['Message'])  # noqa: E501
+            break
+        elif (time.time()-start_time) > max_response_time:
+            raise TimeoutError('Time exceeded the `max_response_time`.')
+        time.sleep(5)  # Sleep for 5 seconds before each data retrieval attempt
+
+    # Extract time series data
+    data = pd.DataFrame(results_json['WeatherDataResults'][0]['WeatherDataPeriods']['WeatherDataPeriods'])  # noqa: E501
+    # Set index and convert to UTC time
+    data.index = pd.to_datetime(data['ObservationTime'])
+    data.index = data.index.tz_convert('UTC')
```

Comment:
Other iotools functions don't convert to UTC; why do it here?

Reply:
I think you are correct good sir!